CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# unreleased
+
+Updating:
+````
+ - zcash_client_backend v0.6.0 -> v0.6.1
+ - zcash_client_sqlite v0.4.0 -> v0.4.2
```

Comment:
Just for future reference, the update to `zcash_primitives 0.9.1` should also have been documented.